include/zephyr/modem/chat.h

```diff
@@ -90,36 +90,46 @@ struct modem_chat_match {
  * @brief Modem chat script chat
  */
 struct modem_chat_script_chat {
-	/** Request to send to modem formatted as char string */
-	const char *request;
+	/** Request to send to modem */
+	const uint8_t *request;
+	uint8_t request_size;
```

Comment:
Please add documentation comment for all fields so that generated documentation looks good.

Reply:
Added comments to all members, and cleaned up some newlines :) See https://github.com/zephyrproject-rtos/zephyr/compare/a0efd2705a0e7532fd30f6c8d1b487bee6f8ab06..dbd6ff2085ef2e30e6887d76d6c73222d9e62458